schemas/src/test/java/com/scottlogic/deg/schemas/ProfileSerialiserTests.java

```diff
@@ -113,6 +113,7 @@ private static RuleDTO createRule(
 
     private static ConstraintDTO createConstraint(Consumer<ConstraintDTO> setupConstraint) {
         ConstraintDTO newConstraint = new ConstraintDTO();
+        newConstraint.is = null;
```

Comment:
Without this - the schema serialiser test failed as we have no current way of serialising an empty object i.e. UndefinedValue()

Reply:
Investigate the use of `@JsonSerialize` to override the method of serialising a property. I.e. can this annotation be used to serialise `is` with a value when its anything other than `undefined` and omit the property AND its value if it is set to `undefined`?

If not then the above change may be our only option, however it wouldn't emit a profile that could then be read.